# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/moz.build
# Commit: c414d82a5325
# Full Hash: c414d82a532508523d40ce63111cb30b2a1076be
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-02-22 05:26:22
# Regressor Bug: 1521879
# File Overlap Count: 1
# Description:
#   Bug 1521879 - Part 1: IPC-based MediaTransport implementation r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D17273
# ==============================================================================

diff -r 3a182b05e2c1 -r c414d82a5325 netwerk/ipc/moz.build
--- a/netwerk/ipc/moz.build	Thu Feb 21 19:01:32 2019 +0200
+++ b/netwerk/ipc/moz.build	Thu Feb 21 16:42:12 2019 +0000
@@ -62,6 +62,8 @@
 LOCAL_INCLUDES += [
     '/caps',
     '/dom/base',
+    '/media/mtransport',
+    '/media/webrtc',
     '/modules/libjar',
     '/netwerk/base',
     '/netwerk/protocol/http',
